Tests/NIOTransportServicesTests/NIOTSBootstrapTests.swift

```diff
@@ -173,6 +173,7 @@ final class NIOTSBootstrapTests: XCTestCase {
             XCTFail("can't connect to server1")
             return
         }
+        XCTAssertNotNil(try client1.getMetadata(definition: NWProtocolTCP.definition).wait() as? NWProtocolTCP.Metadata)
```

Comment:
Can we add some testing of the error cases?

Reply:
Added in https://github.com/apple/swift-nio-transport-services/pull/163/commits/c6bd6ed8c605d384e1bb43f0ba50468530ea4b1d